Amazon SQS Connector in New Template (#379)
mulesoft/docs-connectors: modules/ROOT/pages/amazon/amazon-sqs-connector-xml-maven.adoc

= Amazon SQS XML and Maven Support - Mule 4
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:imagesdir: ../../assets/images/

> If you manually code a Mule app in XML either from the Anypoint Studio XML editor or from a text editor, you can get access to Amazon SQS Connector in your app by adding reference statements to your XML Mule flow and in the Apache Maven `pom.xml` file.

Reviewer: ```suggestion
If you manually code a Mule app in XML either from the Anypoint Studio XML editor or from a text editor, you can get access to Anypoint Connector for Amazon SQS (Amazon SQS Connector) in your app by adding reference statements to your XML Mule flow and in the Apache Maven `pom.xml` file.
```

Reviewer: Also, can we reword one or the other of "adding reference statements to your XML Mule flow" and the subtitle "Add a Namespace to the Connector"? Or is that connection obvious to your audience?

[NOTE]
====

> You can avoid the manual process described in this topic by using Studio. When you install Amazon SQS Connector in Studio, Studio automatically populates the XML code with the Amazon SQS Connector namespace and schema location, and adds a dependency to the `pom.xml` file.

Reviewer: ```suggestion
You can avoid the manual process described next by using Studio. When you install Amazon SQS Connector in Studio, Studio automatically populates the XML code with the Amazon SQS Connector namespace and schema location, and adds a dependency to the `pom.xml` file.
```
Do we have anything to add about WHY a user might PREFER this manual process? Why are we documenting it?

Author: In case they configure a connector in Design Center or in an XML editor other than Studio.

xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core
http://www.mulesoft.org/schema/mule/core/current/mule.xsd
http://www.mulesoft.org/schema/mule/sqs
http://www.mulesoft.org/schema/mule/sqs/current/mule-sqs.xsd
http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
http://www.mulesoft.org/schema/mule/ee/core http://www.mulesoft.org/schema/mule/ee/core/current/mule-ee.xsd">
----

== Add a POM File Dependency

The `pom.xml` Maven file generated by Studio contains dependencies for
a Mule app. When you code a Mule app manually, include this XML snippet in
your `pom.xml` file to enable access to the connector.

The Mule runtime engine translates the `RELEASE` dependency keyword to the current version.

[source,xml,linenums]
----
<dependency>
    <groupId>com.mulesoft.connectors</groupId>
    <artifactId>mule-amazon-sqs-connector</artifactId>
    <version>RELEASE</version>
    <classifier>mule-plugin</classifier>
</dependency>
----

To obtain the most up-to-date `pom.xml` file information, access the
connector in https://www.mulesoft.com/exchange/[Anypoint Exchange]

> and click *Dependency Snippets*.

Reviewer: Is this a prerequisite to "include this XML snippet in your..."?
Should it come earlier in the topic?

Author: Noted for next template update.

== Next Step

After completing your namespace and `pom.xml` file, you can try

> the xref:amazon/amazon-sqs-connector-examples.adoc[Examples].

Reviewer: ```suggestion
the provide xref:amazon/amazon-sqs-connector-examples.adoc[examples].
```

Author: Not sure why you added "provide." I incorporated the other change.
